Pipelines/Templates/run-canvas-tests.yml

```diff
@@ -43,7 +43,10 @@ steps:
       Import-Module $microsoftPowerAppsAdministrationPowerShellModule -Force -RequiredVersion $env:PowerAppsAdminModuleVersion -ArgumentList @{ NonInteractive = $true }
       Add-PowerAppsAccount -TenantID $env:TenantId -ApplicationId $env:ClientId -ClientSecret "$(ClientSecret)"
 
-      Set-CanvasTestAutomationURLs $env:SPNTOKEN $serviceConnection $solutionName $canvasAppsPath $env:EnvironmentId
+      Write-Host "Env EnvironmentId - " $env:EnvironmentId
+      Write-Host "Build EnvironmentId - " $(BuildTools.EnvironmentId)
+      #Set-CanvasTestAutomationURLs $env:SPNTOKEN $serviceConnection $solutionName $canvasAppsPath $env:EnvironmentId
```

Comment:
@RajeevPentyala can we remove the unused code and extra debug logging here?

Reply:
Disregard. I think this is from my branch :)

Reply:
Disregard the last comment will handle this on merge since this was taken from my branch

Reply:
Want to make sure we have this scenario covered in our PR Loop test. Let's add a secret for the first export pipeline run that has the ServiceConnectionName and Url empty and then set the ServiceConnectionName and Url on the second pass with a different secret payload to ensure this is working as expected.